config/settings/production.yml

```diff
@@ -11,3 +11,4 @@ search_ui:
 environment:
   label: "Beta"
   selector_name: "beta"
+allocations_state: open
```

Comment:
does this needs to go to azure.yml

?

Reply:
No, I don't think so. We are going to control this setting ourselves instead of needing azure to control it for us.